README.md

```diff
@@ -38,13 +38,15 @@ Encrypted vars can be added to `.travis.yml` using the travis cli tools as decri
 Currently configured with GITHUB_TOKEN value.
 
 ### To Production
-The production is hosted using (now.sh)[https://now.sh].
+The production is hosted using [now.sh](https://now.sh).
 The process is manual and requires deploying two now apps. You can do that by installing now and deploying the separate now.json files.
 To speed this up you can make sure now is installed globally before running npx `yarn global add now`.
+This will deploy what is currently in your dist folder, so make your have a build in `dist/`
```

Comment:
Typo in `so make your`

Reply:
```suggestion
This will deploy what is currently in your dist folder, so make sure you have a build in `dist/` that you want to deploy.
```

Reply:
Cool new button!
![image](https://user-images.githubusercontent.com/32434620/66407400-0c05cc80-e9e5-11e9-953f-7b784b604136.png)

Yep that reads better 👍